aws_sqs_entity.module

```diff
@@ -58,10 +58,15 @@ function aws_sqs_entity_entity_delete($entity, $type) {
  *   The CRUD operation from hook_entity_OPERATION().
  * @param bool $skip_rules
  *   Optionally skip CrudQueue::checkRules().
+ *
+ * @return bool Whether the entity published or not.
```

Comment:
I can see the value of moving `checkRules()` from `getQueue()` to here. Keeping the params simple is nice.

I can also see the value of making this function more robust with the `sendItem()` status bool return, so you can re-use it from elsewhere and get that info for various needs immediately and not just through the notification hook. In this case though do you mind changing the opening comment from 
> Wrapper for our Entity CRUD hook implementations.

to something more generic, like
> AWS SQS Entity CRUD action wrapper.

since if it were just for CRUD hook implementations return status wouldn't be used.

Reply:
👍